### PR TITLE
Copy to clipboard functionality and Grid Layout snippet

### PR DESCRIPTION
**Pull request by fejes713**
Pull request summary: 

1. `Copy to clipboard` button that copies `css` content to users clipboard. It works. Lacks `autoprefixer` support but for now, it is okay. Some people won't want to copy `autoprefixed` css anyways. 

_note:_ The button works but button's placement isn't right. 

2. Grid layout snippet shows basic website layout that can be achieved using `grid`. 

I've run both `parcel` and `build` scripts. **After button correct placement, the branch is ready to merge.**

### CODE REVIEW

**Comment by fejes713** [comment]
I've removed `autoprefixer.min.js` as it had 40.000 lines.

File: src/js/components/ClipboardCopy.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+snippets.forEach(snippet => {
+  const css = snippet.querySelector('pre code.lang-css').textContent
+  const autoprefixCss = css //Hook autoprexier to this part
+
+  const copyButton = document.createElement('button')
+  copyButton.classList = 'btn is-large codepen-btn'
+  copyButton.innerHTML = '<i data-feather="copy"></i>Copy to clipboard'
+  copyButton.onclick = () => copyToClipboard(autoprefixCss)
+
+  snippet.insertBefore(copyButton, snippet.querySelector('h4[data-type="CSS"]').nextSibling)
```
Comment (Chalarangelo) [inline src/js/components/ClipboardCopy.js]: I believe `.nextSibling.nextSibling` would be kind of what you want to achieve here (placing below the code block). Or adding below the `DEMO` section, much like the Codepen button (I like that better in all honesty, but it's mainly personal preference).

**Comment by Chalarangelo** [comment]
Autoprefixing can be left out for now, it might take a while to implement and copying to clipboard is actually a very important feature to get on the website, even if not 100% complete with autoprefixing etc.

**Comment by fejes713** [comment]
@Chalarangelo I will see to fix the button placement tomorrow if no one touches that by then. Its ready to merge then!

File: src/js/components/ClipboardCopy.js (hunk 1 above)
Comment (fejes713) [reply src/js/components/ClipboardCopy.js]: @atomiks wrote that line for me. The button is supposed to go next to the `css` `h4` tag. It is there right now but the spacing and margins aren't right.

**Comment by fejes713** [comment]
@Chalarangelo @atomiks I've fixed the styling. Looks fine to me now (both Desktop and Mobile). Again there are some conflicts, how do I resolve them?

**Comment by fejes713** [comment]
As I've added `Grid` snippet I am deleting this PR. I will push `copyToClipboard` changes later this day

**Pull request closed without merging**